### ISSUE TITLE
What the point of using drop

### ISSUE THREAD

**Issue by KINGLRL** [jeya-maria-jose/UNeXt-pytorch#37]
In your code, I don't see you set the dropout rate > 0.
Did u set it > 0 during your training?
